### ISSUE TITLE
yAxis no longer accept objects

### ISSUE THREAD

**Issue by chripede** [pablojim/highcharts-ng#367]
After upgrading to 0.0.9 this no longer works:

```
yAxis: {
    title: {
        text: null,
    },
    labels: {
        format: "{value}%"
    },
    max: 100,
    tickInterval: 20
}
```

If I add it to yAxis as an array it does work, but according to Highcharts docs this should only be needed if there are multiple axes. http://api.highcharts.com/highcharts#yAxis

**Comment by cedricduffournet** [comment]
I have the same issue. It seems to work by moving yAxis configuration in chartConfig.options :
```code
chartConfig = {
    options: {
        yAxis: {
            title: {
                text: null,
            },
            labels: {
                format: "{value}%"
            },
            max: 100,
            tickInterval: 20
        }
    }
}
```

**Comment by pablojim** [comment]
ok I'll have a look at this. Suggest sticking with 0.0.8 for now.

**Comment by bakurin** [comment]
It seems there is an error in function 
```javascript
deepExtend(destination, source)
```
error occurs when `source` var is an object and `destination` is undefined

**Comment by TheSharpieOne** [comment]
I didn't see this ticket when I created a PR... so I'll put this information here which better documents the issue:

`TypeError: Cannot read property '<PROPERTY>' of undefined`
where the destination is `undefined`.
[this recent change](https://github.com/pablojim/highcharts-ng/compare/0.0.8...0.0.9#diff-415045283d3b09f7aaee8d0dac532f12R175) from 0.8 to 0.9 which uses `deepExtend` instead of `angular.copy` is what is triggering the issue in the recent change (since `deepExtend` did not change).

Breaking it down:
[This recent change](https://github.com/pablojim/highcharts-ng/compare/0.0.8...0.0.9#diff-415045283d3b09f7aaee8d0dac532f12R161) adds a default for xAxis, but no default for yAxis was added.  Not supplying one in the `config.options` will cause `mergedOptions[axisName]` to be `undefined` when `axisName` is `'yAxis'` (which it gets from [this array](https://github.com/pablojim/highcharts-ng/compare/0.0.8...0.0.9#diff-415045283d3b09f7aaee8d0dac532f12R141)). When it tries to deepExtend the `config.yAxis` on to that, this issue occurs.
